Dockerfile

```diff
@@ -1,7 +1,40 @@
-FROM domerd/marine-core:marine
+ARG MARINE_CORE_TAG=marine
 
-RUN python3.8 -m pip install tox
+FROM domerd/marine-core:$MARINE_CORE_TAG as builder
 
 WORKDIR /marine-python
 
-CMD tox
+COPY setup* ./
+COPY README* ./
+COPY LICENSE ./
+COPY marine ./marine
+
+ENV PY="/opt/python/cp38-cp38/bin/python"
```

Comment:
Can you embed the python version in the `marine-core` image name or tag name instead?

Reply:
That's not really relevant, the manylinux image is bundled with python 3.5 through 3.9.
I just chose to use 3.8 here cause why not.